gen/gen.go

```diff
@@ -83,7 +83,7 @@ func Gen(exercise string, j interface{}, t *template.Template) error {
 		return err
 	}
 	// write output file for the Go test cases.
-	return ioutil.WriteFile(filepath.Join(dirProblem, "cases_test.go"), src, 0777)
+	return ioutil.WriteFile(filepath.Join(dirProblem, "cases_test.go"), src, 0666)
```

Comment:
I can make it 0644 if someone really cares, though I expect people's umasks to mask out group write/world write, so I leave it as 0666. To wit, on my machine the umask is 077, so the file gets created 0600.